packages/react-core/src/components/AboutModal/AboutModal.tsx

```diff
@@ -6,7 +6,7 @@ import { AboutModalBoxHeader } from './AboutModalBoxHeader';
 import { AboutModalBoxBrand } from './AboutModalBoxBrand';
 import { AboutModalBoxCloseButton } from './AboutModalBoxCloseButton';
 import { AboutModalBox } from './AboutModalBox';
-import { Modal, ModalVariant } from '../Modal';
```

Comment:
we should be using the new modal here.  I am ok with follow up issue to do that, but it should be done for Beta.